Dockerfile.windows

```diff
@@ -0,0 +1,73 @@
+####################################################################################################
```

Comment:
Can this be build on Linux?

Reply:
Unfortunately no but like mentioned in the PR description we could use AppVeyor, GitHub Workflows or something similar for building the windows image.